.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Sonar Analysis
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+jobs:
+  build:
+    name: Analyse Module with SonarCloud
+    runs-on: ubuntu-latest
```

Comment:
I think we have a lighter linux distribution for docker: may be Alpine

Reply:
done

Reply:
We should respect the PRD

Reply:
I not sure to get in what this not respect the PRD ? 
It is the image used to build the PR (or the branch) not to run exo